components/screens/userProfile/UserProfileScreen.tsx

```diff
@@ -174,9 +69,17 @@ function UserProfileScreen({ route, navigation }: IProps) {
         scrollEnabled={false}
         ref={pagerView}
```

Comment:
curious: what is this `pagerView` ref for? I see we assign it, and then we prop drill it all the way through, but I don't understand what we use it for, if anything. Can it just be removed?